hail_scripts/v02/utils/elasticsearch_client.py

```diff
@@ -42,6 +42,7 @@ def export_table_to_elasticsearch(
         export_globals_to_index_meta=True,
         verbose=True,
         write_null_values=False,
+        elasticsearch_config={},
```

Comment:
its really bad practice to use list or dict literals as default kwargs- I'm surprised the linter isn't complaining. Its beeter to use None as the default and then handle it in the code (i.e `elasticsearch_config = elasticsearch_config or {}`